conda.yaml

```diff
@@ -6,10 +6,10 @@ dependencies:
   # Define conda packages here.
   # If available, always prefer the conda version of a package, installation will be faster and more efficient.
   # https://anaconda.org/search
-  - python=3.7.5
+  - python=3.9.13
 
-  - pip=20.1
+  - pip=22.1.2
   - pip:
     # Define pip packages here.
     # https://pypi.org/
-    - rpaframework==10.6.0 # https://rpaframework.org/releasenotes.html
+    - rpaframework==17.4.0 # https://rpaframework.org/releasenotes.html
```

Comment:
Why not jumping straight to the latest: **19.x.x**

Reply:
Because I matched our [standard template](https://github.com/robocorp/template-standard) which is what Kari told me was how we should be updating portal bots.

Reply:
Good to know, thx!